fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1865,4 +1865,23 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         appContext.getPreferenceKey(R.string.pref_key_hidden_engines_restored),
         default = false,
     )
+
+    /**
+     * Indicates if the Firefox Suggest feature is enabled.
+     */
+    var enableFxSuggest by lazyFeatureFlagPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_enable_fxsuggest),
+        default = { FxNimbus.features.fxSuggest.value().enabled },
+        featureFlag = true,
+    )
+
+    var showSponsoredSuggestions by booleanPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_show_sponsored_suggestions),
+        default = false,
```

Comment:
That's right! If they're in the experiment, they'll _see_ the settings to show sponsored and non-sponsored suggestions, and we'll fetch them—but defaulting to `false` here means they'll still have to turn them on manually to see them, which isn't what I want. Thanks for catching that!

Do you think I could write this (and for `showNonSponsoredSuggestions`) as:

```suggestion
        default = enableFxSuggest,
```

...Since `enableFxSuggest` already checks whether the experiment or the override pref is set?

Reply:
Yes, that works too!